cmd/login.go

```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context'.")
```

Comment:
I would add a link to:
https://okteto.com/docs/reference/cli/#context

And I would say it's is deprecated in favor of `okteto context`. And it won't be available anymore from 1.16+

Reply:
Oh, great.